### PR TITLE
Release process for version 0.1

### PR DESCRIPTION
**Pull request by frreiss**
This PR contains changes from going through the [release checklist](https://github.com/CODAIT/text-extensions-for-pandas/blob/master/docs/release.md) prior to the 0.1 release.

Main changes:
* Increment version number in `setup.py`
* Update environment config so that all tests and notebooks run out of the box after running `env.sh`
* Rerun all notebooks in the `notebooks` directory and update outputs as needed
* Rerun all notebooks in the `tutorials` directory and update outputs as needed

### CODE REVIEW

**Comment by frreiss** [comment]
@BryanCutler I'm going to do CTR on this PR so as not to block the other parts of the release process. Can you please have a look at these changes when you have a moment?

File: notebooks/Analyze_Model_Outputs.ipynb [hunk 1]
```diff
@@ -844,83 +844,736 @@
      "data": {
```
Comment (BryanCutler) [inline notebooks/Analyze_Model_Outputs.ipynb:844]: <p>Is this supposed to be removed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CODAIT/text-extensions-for-pandas/pull/229/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

**Review by BryanCutler — COMMENTED** [review]
LGTM, there are some cells in the NBs that look like they are missing. Not sure if that is intended or because of an issue with the css file?

File: notebooks/Analyze_Model_Outputs.ipynb (hunk 1 above)
Comment (frreiss) [reply notebooks/Analyze_Model_Outputs.ipynb:844]: <p>That's strange, it looks ok on my machine and on Github's own renderer. I guess ReviewNB is mangling HTML output in a way that we didn't expect. From a quick poke at the DOM tree for this page, it looks like ReviewNB is trying to run the JavaScript in the visualization and ending up with an empty div somehow.</p><p> </p><p>FYI @<a href="https://github.com/PokkeFe" target="_blank">PokkeFe</a></p>

**Pull request merged**